jax/random.py

```diff
@@ -383,6 +383,34 @@ def _normal(key, shape, dtype):
   return onp.array(onp.sqrt(2), dtype) * lax.erf_inv(u)
 
 
+def truncated_normal(key, lower, upper, shape=(), dtype=onp.float64):
+  """Sample truncated standard normal random values with given shape and float
+  dtype.
+
+  Args:
+    key: a PRNGKey used as the random key.
+    lower: a lower bound for truncation.
```

Comment:
Can you give more type-ish information here? Like, does it have to be a scalar? Should we say that it should be of floating dtype (even if we might be looser and up-cast integers)?